openshift/che-monitoring.yaml

```diff
@@ -128,6 +44,11 @@ objects:
               port: 9090
             initialDelaySeconds: 30
             timeoutSeconds: 30
+          resources:
+            limits:
+              memory: 1Gi
```

Comment:
Since this is for a single service, can we start small? perhaps set it to 256Mi? 

If there's some benchmarking alreay done for this locally, please post the results here.

Reply:
Set to 256Mi. No benchmarks made. Just made a small observation.  If I'm not mistaken Prometheus process takes ~1-10Mi